daccs_client/client.py

```diff
@@ -31,94 +25,96 @@ def __init__(self, fallback: Optional[bool] = True) -> None:
             if the cloud registry cannot be accessed, defaults to True
         :type fallback: Optional[bool], optional
         :raises requests.exceptions.RequestException: Raised when there is an issue
-            conencting to the cloud registry and `fallback` is False
-        :raises UserWarning: Raised when there is an issue conencting to the cloud registry
+            connecting to the cloud registry and `fallback` is False
+        :raises UserWarning: Raised when there is an issue connecting to the cloud registry
             and `fallback` is True
         :raise RuntimeError: If cached registry needs to be read but there is no cache
         """
         self._fallback = fallback
-        self._using_fallback = False
         self._nodes: dict[str, DACCSNode] = {}
+        self._registry: dict = {}
 
         try:
-            self._registry = requests.get(REG_URL)
```

Comment:
the logic here was sound but a little more complex than needed (I was getting warnings about variables definitions and other things). This should be simpler and do the same thing